docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -1,280 +1,366 @@
-# Pelorus Demo
+# Quickstart Tutorial
 
-**NOTE** This demo is based on the Pelorus Helm chart installation which is now deprecrated.  Please also refer to the [Pelorus Operator demo](https://github.com/konveyor/pelorus/tree/master/demo)  
-
-For Pelorus installation to monitor your application's workflow, please skip this Demo and jump straight to the [Installation](../Installation/) and [Configuration](../configuration/PelorusCore/) part of the documentation.
-
-In this demo, you will
+In this Quickstart you will:
 
 * Get a taste of how Pelorus captures a change going through the application's delivery cycle.
 * Understand that Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
-* Use GitHub as the Hosting Service for Version Control System (used by commit time exporter) and as the Issue Tracker System (used by failure exporter) (a GitHub account is necessary).
+* Use GitHub as the Hosting Service for Version Control System (used by commit time exporter) and as the Issue Tracker System (used by failure exporter).
 
 To do so, you will
 
+* Install Pelorus Operator in the OpenShift cluster in the `pelorus` namespace
+* Create Pelorus instance to monitor your sample application
```

Comment:
I would change "your" to "a", because I am afraid it can pass the message that user already needs one of his own

Reply:
changed.